nuclei/CVE-2022-29499.yaml

```
@@ -1,27 +1,58 @@
 id: CVE-2022-29499
 
 info:
-  name: Mitel MiVoice Connect Remote Code Execution
+  name: Mitel MiVoice Connect - Remote Code Execution
   author: dmartyn
   severity: critical
   description: |
-    The Service Appliance component in Mitel MiVoice Connect through 19.2 SP3 allows remote code execution because of incorrect data validation. The Service Appliances are SA 100, SA 400, and Virtual SA.
+    Mitel MiVoice Connect through 19.2 SP3 contains a remote code execution vulnerability in the Service Appliance component (SA 100, SA 400, and Virtual SA). The exploit chain involves two steps:
+    1. First request targets vtest.php with a get_url parameter that points to a local ucbsync.php file
+    2. Second request (originating from localhost) to ucbsync.php contains command injection via the syncfile parameter
+    
+    This vulnerability allows attackers to execute arbitrary commands on the affected system. The exploit is particularly dangerous as it bypasses restrictions that prevent direct external access to the vulnerable endpoint.
   reference:
     - https://www.mitel.com/en-ca/support/security-advisories/mitel-product-security-advisory-22-0002
     - https://www.crowdstrike.com/blog/novel-exploit-detected-in-mitel-voip-appliance/
     - https://arcticwolf.com/resources/blog/lorenz-ransomware-chiseling-in/
+    - https://nvd.nist.gov/vuln/detail/CVE-2022-29499
+  classification:
+    cvss-metrics: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H
```

From NVD they put scope as Unchanged..
```suggestion
    cvss-metrics: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
```